Kernel/Devices/Generic/RandomDevice.cpp

```diff
@@ -40,9 +40,12 @@ ErrorOr<size_t> RandomDevice::read(OpenFileDescription&, u64, UserOrKernelBuffer
     });
 }
 
-ErrorOr<size_t> RandomDevice::write(OpenFileDescription&, u64, UserOrKernelBuffer const&, size_t size)
+ErrorOr<size_t> RandomDevice::write(OpenFileDescription&, u64, UserOrKernelBuffer const& buffer, size_t size)
 {
-    // FIXME: Use input for entropy? I guess that could be a neat feature?
+    Bytes bytes {};
```

Comment:
This is still incorrect, isn’t it?
You still don’t own that ‘Bytes’ buffer because you did not allocate properly.

Reply:
uhh what is the correct way?